docs/docs-content/release-notes/release-notes.md

```diff
@@ -11,9 +11,176 @@ tags: ["release-notes"]
 
 <ReleaseNotesVersions />
 
-## Sep 14, 2024 - Release 4.4.X
+## Sep 14, 2024 - Release 4.4.15
 
-TBA
+### Palette Enterprise {#release-4-4-15}
+
+##### Features
+
+- Palette will not remove pods with the toleration key node.kubernetes.io/unschedulable set to NoSchedule. This is
+  useful for scenarios where a pod may behave as a DaemonSet but is not a DaemonSet, such as the Portworx operator pod,
+  and you don't want it to be removed during node drain operations. Refer to the
+  [Pod Drainage Toleration](../clusters/cluster-management/node-pool.md#pod-drainage-toleration) section for more
+  information on using the new behavior.
```

Comment:
@karl-cardenas-coding Could we add something about Kevin's suggestion [here](https://spectrocloud.atlassian.net/browse/PCP-2902?focusedCommentId=97208)? `This needs to come with a release note, informing customers using Portworx to set a node repave interval on their worker pool to ensure sufficient time is allotted for Portworx to initialize on a newly repaved cluster node.`

Reply:
@cloudmaniac That's better placed in the README. We can add a short note and link to the README, where a more detailed explanation can be added. I'll defer to you and @kreeuwijk for updating the pack README.